packages/dropdownable/components/Dropdownable.tsx

```diff
@@ -24,14 +24,14 @@ const DEFAULT_DIRECTION_PREFERENCES = [
 
 const Container = styled("div")`
   position: relative;
-  display: inline-block;
 `;
 
 export interface DropdownableProps {
   open: boolean;
   dropdown: React.ReactElement<any>;
   preferredDirections?: Direction[];
-  onClose: () => void;
+  onClose?: () => void;
```

Comment:
@brandonc - the Typeahead component doesn't need to manage the state for whether the dropdown menu is visible. I'm using render props from [downshift](https://github.com/downshift-js/downshift) to update the `open` prop

If you want `onClose` to be required when using `Dropdownable`, I can take a different path